packages/api/cms-api/src/user-permissions/access-control.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from "@nestjs/common";
+
+import { CurrentUserInterface } from "../auth/current-user/current-user";
+import { ContentScope } from "./interfaces/content-scope.interface";
+import { AccessControlServiceInterface } from "./user-permissions.types";
+
+@Injectable()
+export class AccessControlService implements AccessControlServiceInterface {
```

Comment:
I was kinda confused why's there an AccessControlService in both the application and the library. Should we name this DefaultAccessControlService to avoid confusion?

Reply:
I'm not happy either about the naming. @nsams What do you think about the suggestion (`DefaultAccessControlService`)?
Especially with https://github.com/vivid-planet/comet/pull/1380 which implies the the libraries-AccessControlService will be overwritten regularily.

Reply:
Let's continue the discussion here: https://github.com/vivid-planet/comet/pull/1358#discussion_r1390933519